Virtusize/Tests/APIRequestTests.swift

```diff
@@ -178,28 +179,22 @@ class APIRequestTests: XCTestCase {
             ]
             ]
         )
-        XCTAssertEqual(
-            (actualParams?.additionalInfo["modelInfo"]?.value as? [String: Any])?["size"] as? String,
-            "38"
-        )
-        XCTAssertEqual(
-            (actualParams?.additionalInfo["modelInfo"]?.value as? [String: Any])?["hip"] as? Int,
-            85
-        )
-        XCTAssertEqual(actualParams?.additionalInfo["brand"]?.value as? String, "Virtusize")
-        XCTAssertEqual(actualParams?.additionalInfo["fit"]?.value as? String, "regular")
+        
+       
+        XCTAssertEqual(actualParams?.items.first?.additionalInfo["brand"]?.value as? String, "Virtusize")
+        XCTAssertEqual(actualParams?.items.first?.additionalInfo["fit"]?.value as? String, "regular")
         XCTAssertEqual(actualParams?.bodyData.count, 22)
         XCTAssertEqual((actualParams?.bodyData["chest"])?["value"]?.value as? Int, 755)
-        XCTAssertEqual(actualParams?.itemSizesOrig.count, 3)
-        XCTAssertEqual(actualParams?.itemSizesOrig["36"]?["bust"], 645)
+        XCTAssertEqual(actualParams?.items.first?.itemSizesOrig.count, 3)
+        XCTAssertEqual(actualParams?.items.first?.itemSizesOrig["36"]?["bust"], 645)
 		XCTAssertEqual(actualParams?.userGender, "female")
```

Comment:
If possible `female` enum (in all files)

Reply:
This is the test case for request body to check we are sending correct body and all value will be dynamic to check various datatypes, or other values

Reply:
Understood thanks